### PR TITLE
feat: HAR <> Firecamp request converter

### PR DESCRIPTION
**Pull request by jatingodnani**
#115 
hey @Nishchit14,I added code, can you tell me if what I'm doing is correct .you can  tell me hat i am doing wrong

### CODE REVIEW

**Comment by jatingodnani** [comment]
@Nishchit14 already sign cla license then why it showing incorrect

**Comment by Nishchit14** [comment]
follow this steps

1. sign cla (your commit username is different than your GitHub username, please read the description given by the cla bot)
2. follow camelcase naming conventions, check all spelling related comments by spell bot
3. set up tests with expected behavior, following the TDD approach here. (you can search on the web HAR to X GitHub repo to find some references.)

**Comment by jatingodnani** [comment]
Screenshot 2023-10-21 135341
i think name is same

**Comment by Nishchit14** [comment]
@jatingodnani 

image

**Comment by jatingodnani** [comment]
@Nishchit14 i will if completed,i am asking you to check my approch because i havent make any converter

File: packages/firecamp-har/src/index.ts [hunk 1]
```diff
@@ -1,12 +1,72 @@
...
 import { Collection } from '@firecamp/types';
 
+
 const firecampToHar = (collection: Partial<Collection>): any => {
   return {};
 };
 
 const harToFirecamp = (har: Har): Partial<Collection> => {
-  return { requests: [] };
+  const exctractQuerParams=(queryparams:Har.queryString[])=>{
```
Comment (jsoref) [inline packages/firecamp-har/src/index.ts:10]:
```suggestion
  const extractQueryParams=(queryParams:Har.queryString[])=>{
```
?

File: packages/firecamp-har/src/index.ts [hunk 2]
```diff
@@ -1,12 +1,72 @@
...
+    if(queryparams.length===0){
+      return []
+    }
+    const convertesqueryparams=queryparams.map((obj)=>{
```
Comment (jsoref) [inline packages/firecamp-har/src/index.ts:14]:
```suggestion
    const convertedQueryParams=queryparams.map((obj)=>{
```
?

**Pull request closed without merging**